client/disputes/test/__snapshots__/index.tsx.snap

```diff
@@ -293,7 +293,7 @@ exports[`Disputes list renders correctly 1`] = `
                 >
                   <a
                     data-link-type="external"
-                    href="http://test.local/order/1"
+                    href="post.php?post=1&action=edit"
```

Comment:
why we have different snapshoots ? shouldn't be the same as before ?

Reply:
Good point, need to have a look